plugins/woocommerce/tests/e2e-pw/envs/default-pressable/playwright.config.js

```diff
@@ -7,7 +7,10 @@ config = {
 		{
 			name: 'default pressable',
 			use: { ...devices[ 'Desktop Chrome' ] },
-			testMatch: '**basic.spec.js',
+			testMatch: [
+				'**/basic.spec.js',
+				'**/activate-and-setup/**/*.spec.js',
```

Comment:
As we update new tests, we will add them here to run regularly until all tests are revised and passed.